filebeat/tests/system/test_json.py

```diff
@@ -64,40 +64,6 @@ def test_docker_logs_filtering(self):
         assert all(o["stream"] == "stdout" for o in output)
         assert all("windows" not in o["log"] for o in output)
 
-    def test_docker_logs_multiline(self):
-        """
-        Should be able to do multiline on docker logs.
-        """
-        self.render_config_template(
```

Comment:
@kvch 

* I think this test is still valuable to make sure we support multiline logs in docker. What we should change is to set `keys_under_root=False` and then check for the fields accordingly
* If we remove the test, we should also remove the test file as it's not used anywhere else.

Reply:
The "official" was has become the docker input I would say. This doesn't mean users do not still use the old behaviour and I'm not even sure if we have such a multiline test for the docker input.